script.js

```diff
@@ -1,12 +1,142 @@
-}
+  let allShow = getAllShows();
+  //Sorts shows in alphabetical order
+  allShow.sort(function (a, b) {
+    if (a.name < b.name) {
+      return -1;
+    }
+    if (a.name > b.name) {
+      return 1;
+    }
+    return 0;
+  });
 
-function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
-}
+    const homebtn = document.createElement("button");
+    homebtn.textContent = "Home";
+    homebtn.className = "home";
+    const search = document.querySelector("#search");
+    const searchInput = document.createElement("input");
+    const episodeNum = document.createElement("p");
+    search.appendChild(homebtn);
+    //displayPage(episodeList);
+    let selectShow = document.createElement("select");
+    selectShow.className = "selectshow";
+    search.appendChild(selectShow);
+    let selectEpisode = document.createElement("select");
+    selectEpisode.className = "selectepisode";
+    search.appendChild(selectEpisode);
+    searchInput.type = "text";
+    searchInput.placeholder = "Search";
+    search.appendChild(searchInput);
+    search.appendChild(episodeNum);
+    episodeNum.style.marginLeft = "30px";
+    
+    //This fetch calls the first object in (alphabetical) order and displays the main page
+    fetch(`https://api.tvmaze.com/shows/${allShow[0].id}/episodes`)
+    .then((response) => response.json())
+    .then((data) => {
+      displayPage(data);
+      oneShow(data);
+      displayEpisode(data);
+      searchTerm(data)
+    })
+    .catch((error) => console.log(error));
+
+//Makes search on episodes of selected show
+    function searchTerm (array) {
+      searchInput.addEventListener("keyup", function (e) {
+        e.preventDefault();
+        let term = searchInput.value.toLowerCase();
+        let filterEpisode = array.filter(
+          (element) =>
+            element.name.toLowerCase().includes(term) ||
+            element.summary.toLowerCase().includes(term)
+        );
+        //This part displays the searched elements
+        displayPage(filterEpisode);
+        episodeNum.textContent = `${filterEpisode.length} episode(s)`;
+      })
+    }
+    
+    //Creates the tags and set up the HTML elements
+    function displayPage(episodeList) {
+      const container = document.getElementById("container");
+      container.innerHTML = "";
+      episodeList.forEach((element) => {
+        const episode = document.createElement("div");
+        const title = document.createElement("h3");
+        const img = document.createElement("img");
+        const summary = document.createElement("p");
+        episode.className = "episode";
+        title.className = "title";
+        img.className = "image";
+        summary.className = "summary";
+        title.innerHTML = `${element.name} - S${element.season < 10 ? 0 : ""}${
+          element.season
+        }E${element.number < 10 ? 0 : ""}${element.number}`;
```

Comment:
You may want to use padStart here as another option. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart